main/iptables/APKBUILD

```diff
@@ -2,14 +2,14 @@
 
 pkgname=iptables
 pkgver=1.8.2
```

Comment:
1.8.3 is out: https://www.netfilter.org/projects/iptables/files/changes-iptables-1.8.3.txt